### PR TITLE
Change loglevel of extensions to debug if content is empty and rc is 0

### PR DESCRIPTION
**Pull request by runningman84**
This reduces spam like this
```
{"timestamp":"2017-09-08T21:55:51.732779+0000","level":"info","message":"handler extension output","extension":{"type":"extension","name":"influxdb"},"event":{"id":"e83b95b3-9fbb-4797-9761-3eb1c2c7cb33"},"output":"","status":0}
```

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by majormoses** [comment]
seems reasonable

**Comment by majormoses** [comment]
regarding the build failure my guess is that we need to update bundler installed in travis. Something like: https://github.com/sensu-plugins/sensu-plugins-skel/blob/master/.travis.yml#L4-L5

File: lib/sensu/server/handle.rb [hunk 1]
```diff
@@ -126,12 +126,21 @@ def transport_handler(handler, event_data)
       # @param event_id [String] event UUID
       def handler_extension(handler, event_data, event_id)
         handler.safe_run(event_data) do |output, status|
-          @logger.info("handler extension output", {
-            :extension => handler.definition,
-            :event => { :id => event_id },
-            :output => output,
-            :status => status
-          })
+          if output == '' and status == 0
```
Comment (portertech) [inline lib/sensu/server/handle.rb]: How about this instead?

```
log_level = (output == "" && status == 0 ? :debug : :info 
@logger.send(log_level, "handler extension output", {
...
})
```

Comment (portertech) [reply lib/sensu/server/handle.rb]: This pattern is already used elsewhere, https://github.com/sensu/sensu/blob/9059bf10125221cd4806cc0ad6758f43fe216e8b/lib/sensu/server/process.rb#L258-L259

Comment (majormoses) [reply lib/sensu/server/handle.rb]: Only slight tweaks to make it more ruby:
```ruby
log_level = (output.empty? && status.zero?) ? :debug : :info 
@logger.send(log_level, "handler extension output", {
...
})
```

Comment (runningman84) [reply lib/sensu/server/handle.rb]: ok I have modified the pull request...

**Comment by portertech** [comment]
The build failures are due to the version of Bundler and a know touchy test that needs to be rewritten 👍

**Comment by portertech** [comment]
@runningman84 Thank you! This will be in the next patch release 👍 

50-caliber-rambo-kitty-assault-unicorn

**Pull request merged**